client/my-sites/upgrades/domain-management/edit-contact-info/form-card.jsx

```diff
@@ -152,6 +170,20 @@ class EditContactInfoFormCard extends React.Component {
 		);
 	}
 
+	renderBackupEmail() {
+		const currentEmail = this.props.contactInformation.email,
+			wpcomEmail = this.props.currentUser.email,
+			strong = <strong />;
+
+		return <p>{ this.props.translate(
+			'If you don’t have access to {{strong}}%(currentEmail)s{{/strong}}, ' +
+			'we will also email you at {{strong}}%(wpcomEmail)s{{/strong}}, as backup.', {
+				args: { currentEmail, wpcomEmail },
+				components: { supportLink: <a href={ support.CALYPSO_CONTACT } />, strong }
```

Comment:
Actually, `supportLink` is not used anymore in that string, so it can be removed 🎉